wagtail/admin/api/actions/publish.py

```diff
@@ -0,0 +1,59 @@
+from django.core.exceptions import ValidationError as DjangoValidationError
+from django.shortcuts import get_object_or_404
+from rest_framework import fields, status
+from rest_framework.exceptions import ValidationError
+from rest_framework.response import Response
+from rest_framework.serializers import Serializer
+
+from wagtail.api.v2.utils import BadRequestError
+from wagtail.core.actions.publish_page_revision import PublishPageRevisionAction
+
+from .base import APIAction
+
+
+class PublishPageAPIActionSerializer(Serializer):
+    changed = fields.BooleanField(default=True, required=False)
```

Comment:
I think this will always be ``False`` for a publish action on the API. Since it wouldn't be possible to edit and publish the page with the same request like it is in the page editor.

I'm not sure if it's correct for Wagtail editor to ever set this to ``True`` either since it should emit an "edit" log with the "publish" log if the content was edited when it was published.

Reply:
It makes sense.